kola/tests/bpf/bpf.go

```diff
@@ -0,0 +1,126 @@
+// Copyright The Mantle Authors.
+// SPDX-License-Identifier: Apache-2.0
+package bpf
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/coreos/pkg/capnslog"
+	"github.com/flatcar-linux/mantle/kola/cluster"
+	"github.com/flatcar-linux/mantle/kola/register"
+	"github.com/flatcar-linux/mantle/util"
+)
+
+// cmdPrefix is a temporary hack to pull `bcc` tools into Flatcar
+const cmdPrefix = "docker run -d --name %s -v /lib/modules:/lib/modules -v /sys/kernel/debug:/sys/kernel/debug -v /sys/fs/cgroup:/sys/fs/cgroup -v /sys/fs/bpf:/sys/fs/bpf --privileged --net host --pid host quay.io/iovisor/bcc %s"
+
+var (
+	plog = capnslog.NewPackageLogger("github.com/flatcar-linux/mantle", "kola/tests/bpf")
+)
+
+// Log defines the standard log format
+// from Docker
+// https://docs.docker.com/config/containers/logging/json-file/
+type Log struct {
+	Log    string `json:"log"`
+	Time   string `json:"time"`
+	Stream string `json:"stream"`
+}
+
+func init() {
+	register.Register(&register.Test{
```

Comment:
On arm this fails with:
```
cluster.go:117: WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```
Could you disable this test for arm for the time being?

Reply:
Mm, Docker image has not yet `arm64` support - let's open a tracking issue upstream and exclude ARM64 for this test then.
Thanks for the report !